content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,95 @@
+
+### Log Level
+
+The default logging verbosity for `rippled` in the `log_level` field is set to `debug` and requires several GB or more per day, depending on transaction volumes and client activity. Setting this to a lower level, such as warning, greatly reduces the space and I/O write requirements. The trade-off to a lower setting is decreased visibility for trouble-shooting problems.
+
+
+## Network and Hardware
+
+Each `rippled` server in the XRP Ledger network performs all of the transaction processing work of the network. It is unknown how soon volumes will approach maximum network capacity. Therefore, the baseline hardware for production `rippled` servers should be similar to that used in Ripple's [performance testing](https://ripple.com/dev-blog/demonstrably-scalable-blockchain/).
+
+### Recommendation
+
+For best performance in enterprise production environments, Ripple recommends running `rippled` on bare metal, with the following characteristics:
+
+- Operating System: Ubuntu 16.04+
+- CPU: Intel Xeon 3+ GHz processor with 4 cores and hyperthreading enabled
+- Disk: SSD
+- RAM: 32 GB RAM
+- Network: Enterprise data center network with a gigabit network interface on the host
+
+#### SSD Storage
+
+SSD Storage should support several thousand both read and write IOPS. Maximum read IOPS come from heavily used servers such as those behind https://s1.ripple.com and https://s2.ripple.com, which service transaction and ledger history retrieval requests. This is sometimes over 10,000 reads per second! Maximum write IOPS have come from performance testing in which up to 7,000 writes/s have been observed.
+
+#### CPU Utilization and Virtualization
+Ripple perfmance engineering finds that bare metal servers acheive maximum throughput. However, it is likely that hypervisors would cause minimal degradation in performance.
+
+#### Network
+
+Any enterprise or carrier-class data center should have plenty of network bandwidth to support running `rippled` servers. The minimum requirements are roughly 2Mbps transmit and 2Mbps receive for current transaction volumes. However, these can burst up to 100MBps transmissions when serving up historical ledger and transaction reports. When a `rippled` server initially starts up, it can burst to well over 20Mbps receive.
+
+#### Storage
+
+Ripple recommends estimating storage sizing at roughly 12GB per day of data kept online with NuDB. RockDB requires around 8GB per day. However, future storage requirements are not known and extra capacity should be available to account for future growth. Currently, a server with all XRP Ledger history requires 6.8TB.
+
+<!-- {# ***TODO: Update the dated storage consideration above, as needed. ***#} -->
+<!-- {# ***TODO: Create historic metrics that a user can use to derive what will be required. For ex, a chart with 1TB in 2014, 3TB in 2015, 7TB in 2018 ***#} -->
```

Comment:
I like this. Do we have a JIRA task to do this work for the years the XRP Ledger has existed so far?

Reply:
We do now, @mDuo13, see https://ripplelabs.atlassian.net/browse/DOC-1331 (Ripple only).